client/signup/config/steps-pure.js

```diff
@@ -683,8 +683,8 @@ export function generateSteps( {
 			apiRequestFunction: setDesignOnSite,
 			delayApiRequestUntilComplete: true,
 			dependencies: [ 'siteSlug' ],
-			providesDependencies: [ 'isFSEActive', 'selectedDesign', 'selectedSiteCategory' ],
-			optionalDependencies: [ 'isFSEActive', 'selectedDesign', 'selectedSiteCategory' ],
```

Comment:
Because we're no longer redirecting to the site editor for sites with block themes, there's no need to check if the site has a block theme (is FSE active).

Also, all themes in the design picker support FSE now, so regardless, we don't need this check.

Reply:
The `difm-design-setup-site` also uses the same step component, so can you please make these changes for that step too? Thanks!
https://github.com/Automattic/wp-calypso/blob/c1aa4d616a76297ab390c59e0a490a9e2376492f/client/signup/config/steps-pure.js#L721-L739